frontend/src/components/auth/Signup.tsx

```diff
@@ -36,7 +36,7 @@ const Signup = (): React.ReactElement => {
   };
 
   if (authenticatedUser) {
-    return <Redirect to={HOME_PAGE} />;
+    return <Redirect to={WELCOME_PAGE} />;
```

Comment:
and I think this should be `HOME_PAGE`?
After Harishan takes away the `setAuthenticatedUser` call in this component, `authenticatedUser` should only be set when someone's logged in - so we can send them to Home

Reply:
i think i had them swapped when i was fixing the merge conflicts